docs/src/selectors.md

```diff
@@ -503,34 +503,68 @@ await page.TextContentAsync("article:has(div.promo)");
 
 ## Selecting elements matching one of the conditions
 
-The `:is()` pseudo-class is an [experimental CSS pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:is).
-It is a function that takes a selector list as its argument, and selects any element that
-can be selected by one of the selectors in that list. This is useful for writing large
-selectors in a more compact form.
+### CSS selector list
+
+Comma separated list of CSS selectors will match all elements that can be selected by
+one of the selectors in that list. This is useful for writing large selectors in a
```

Comment:
"This is useful..." sentence seems strange now.

Reply:
Removed.